check-copyrights.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Get all golang files in the repo, excluding the vendor dir.
+files=$(find . -path ./vendor -prune -o -name "*go" -type f | grep -v vendor)
+nocopyright=""
+for f in $files
+do
+	# For each file, check if it contains "Copyright" in the first line.
+	exists=$(head -n 1 $f | grep "Copyright")
```

Comment:
we could also confirm the Copyright is up to date (ie git indicates a commit this year and the current year is present in the line).